homeassistant/components/rova/sensor.py

```diff
@@ -110,6 +153,7 @@ def __init__(
         self.data_service = data_service
 
         self._attr_name = f"{platform_name}_{description.name}"
+        self._attr_unique_id = f"{platform_name}_{description.name}"
```

Comment:
```suggestion
        self._attr_unique_id = f"{platform_name}_{description.key}"
```
You don't want to use the name for the unique_id since we want to use translation keys here and then name will be None

Reply:
Would this not result in a breaking change as `description.name` is referring to the values in `SENSOR_TYPES` (e.g., `sensor.[address]_bio` will change to `sensor.[address]_gft`?

Reply:
Unique id != Entity id, so unique id can for example be a serial number, while the entity id just a normal `sensor.device_something`.

But since all these entities are already in the entity registry, it won't change the name (but that's with entities with unique id, that is, so I expect it will change, but we don't consider that a breaking change)